assets/js/main.js

```diff
@@ -83,21 +86,41 @@ if (!_BLF.blockAnalytics) { // set in main.njk
         variant: $thisScript.data('ab-variant')
     };
 
+    // if we're in a test variant, record it
     if (ab.id && ab.variant) {
         ga('set', 'expId', ab.id);
         ga('set', 'expVar', ab.variant);
         cxApi.setChosenVariation(ab.variant, ab.id);
     }
 
+    // track this pageview
     ga('send', 'pageview');
-}
 
-$('.js-track-clicks').on('click', function () {
-    let category = $(this).data('category');
-    let action = $(this).data('action');
-    let label = $(this).data('label');
-    analytics.track(category, action, label);
-});
+    // track interactions with in-page elements
+    $('.js-track-clicks').on('click', function (e) {
```

Comment:
FWIW the official auto-track plugin for Analytics implements this style of declarative event tracking and outbound link tracking so might be worth looking into that to see that does the job and we can avoid writing any custom code: https://ga-dev-tools.appspot.com/autotrack/